source/_components/sensor.zestimate.markdown

```diff
@@ -56,3 +58,7 @@ These attributes are available:
 - Address
 - Currency
 - Amount
+
+Example screenshot 
+
+![image](home-assistant.github.io/source/images/components/zestimate/zestimateexample.png)
```

Comment:
The image inclusion is still invalid. See:

<https://deploy-preview-6678--home-assistant-docs.netlify.com/components/sensor.zestimate/>

An example from the docs on how:
<https://raw.githubusercontent.com/home-assistant/home-assistant.io/next/source/_docs/installation.markdown>